openwisp_controller/subnet_division/migrations/__init__.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth.models import Permission
+
+from ...migrations import create_default_permissions, get_swapped_model
+
+
+def assign_permissions_to_groups(apps, schema_editor):
+    create_default_permissions(apps, schema_editor)
+    operators_and_admins_can_manage = ['subnetdivisionrule']
+    admin_manage_operations = ['add', 'change', 'delete', 'view']
+    operator_manage_operations = ['view']
```

Comment:
@nemesisdesign will you please double check the permissions? 

[From openwisp-ipam](https://github.com/openwisp/openwisp-ipam/blob/03b6a28f7a6cdb1d3d2e3436c7593f3181be94e7/openwisp_ipam/migrations/__init__.py#L19), operators only have permissions to view subnets. And I believe it's logical to maintain that for SubnetDivisionRule.

Reply:
Ensure users with the "operator" group can only view.